master/internal/rm/kubernetesrm/resource_pool.go

```diff
@@ -258,6 +265,7 @@ func (k *kubernetesResourcePool) addTask(ctx *actor.Context, msg sproto.Allocate
 		k.addrToJobID[msg.AllocationRef] = msg.JobID
 		k.groupActorToID[msg.Group] = msg.JobID
 		k.IDToGroupActor[msg.JobID] = msg.Group
+		k.jobIDToRunningPods[msg.JobID] = 0
```

Comment:
See new implementation -- also, for sake of consistency of initializing all parts of kubernetesResourcePool, I'd argue that we should keep this explicit. imo it's minimal storage cost (because we make sure to clean up & delete it at the end).

Reply:
it is totally fine either way so up to you

but the reasoning behind the comment is to try and follow the Go idiom of using the zero value whenever it makes sense, the zero value of the other maps is going to be "" so we can't in those cases.